src/CollisionHandler.cpp

```diff
@@ -21,25 +23,24 @@ sf::Vector2f CollisionHandler::handleTileCollision(const sf::FloatRect & entityA
 			//Determine direction in which to move entity
 			if (xDifference > 0.0f)
 			{
-				newPos.x = intersection.width;
+				movement.x += intersection.width;
 			}
 			else
 			{
-				newPos.x = -intersection.width;
+				movement.x -= intersection.width;
 			}
 		}
 		else
 		{
 			if (yDifference > 0.0f)
 			{
-				newPos.y = intersection.height;
+				movement.y += intersection.height;
 			}
 			else
 			{
-				newPos.y = -intersection.height;
+				movement.y -= intersection.height;
 			}
 		}
+		break;
```

Comment:
I'm failing to understand exactly what you mean by that.  It seemed to work fine for me.  Regardless, my original comments still apply here, so I'd like to get this changed.